tests/framework/utils/k8s_helper.go

```diff
@@ -295,13 +295,19 @@ func (k8sh *K8sHelper) ResourceOperation(action string, manifest string) (string
 
 //DeletePod performs a kubectl delete pod on the given pod
 func (k8sh *K8sHelper) DeletePod(namespace, name string) (string, error) {
-	return k8sh.DeletePods(name, "-n", namespace)
+	args := append([]string{"--grace-period=0", "pod"}, name)
+	if namespace != "" {
+		args = append(args, []string{"-n", namespace}...)
+	}
+	return k8sh.DeleteResourceAndWait(true, args...)
 }
 
 //DeletePods performs a kubectl delete pod on the given pods
-func (k8sh *K8sHelper) DeletePods(pods ...string) (string, error) {
-	args := append([]string{"--grace-period=0", "pod"}, pods...)
-	return k8sh.DeleteResourceAndWait(true, args...)
+func (k8sh *K8sHelper) DeletePods(pods ...string) (msg string, err error) {
+	for _, pod := range pods {
```

Comment:
does this increase the test running length since we are now deleting pods in serial?  I'm thinking this was for reliability reasons, right?

Reply:
Actually the tests speed up a bit because of the flag `--grace-period=0`. Overall, the tests look about the same length in jenkins even though we added the new upgrade test.